qfdmo/admin/acteur.py

```diff
@@ -788,6 +787,32 @@ def get_readonly_fields(self, request, obj=None):
         return []
 
 
+class ToutPropositionServiceInline(NotMutableMixin, BasePropositionServiceInline):
+    model = ToutPropositionService
+
+
+class ToutActeurLabelQualiteInline(NotMutableMixin, admin.StackedInline):
+    model = ToutActeur.labels.through
+    extra = 0
+
+
+class ToutActeurAdmin(NotMutableMixin, BaseActeurAdmin):
+    inlines = [
+        ToutPropositionServiceInline,
+        ToutActeurLabelQualiteInline,
+    ]
+    fields = list(BaseActeurAdmin.fields) + ["parent"]
+
+    def get_inline_instances(self, request, revision_acteur=None):
+        inlines = []
+        if revision_acteur and revision_acteur.is_parent:
+            inlines.append(ToutActeurChildInline(self.model, self.admin_site))
+        else:
+            inlines.append(ToutPropositionServiceInline(self.model, self.admin_site))
+            inlines.append(ToutActeurLabelQualiteInline(self.model, self.admin_site))
+        return inlines
```

Comment:
```suggestion
        if revision_acteur and revision_acteur.is_parent:
            return [ToutActeurChildInline(self.model, self.admin_site)]
        else:
            return [ToutPropositionServiceInline(self.model, self.admin_site), ToutActeurLabelQualiteInline(self.model, self.admin_site)]
```

on peut p-e légèrement simplifier ici